README.md

```diff
@@ -1,2 +1,25 @@
 # slatedb-pulumi
+
 Pulumi scripts to manage SlateDB committer access to various infrastructure resources.
+
+## Pulumi State
+
+The Pulumi state is stored in an S3 bucket called `slatedb-pulumi` stack. [@criccomini](https://github.com/criccomini) created a bucket on Tigris for this purpose. Eventually, we might want to migrate to Pulumi Cloud or AWS S3.
```

Comment:
did you look at the pulumi cloud at all? I've heard people have had bad experiences with using s3 directly (the state backend was just very slow), and their management ui is quite nice. Their free tier should be fine for this project (up to 200 resources). They also let open source projects apply for free - happy to apply for us.

Reply:
Ya, I did. I found it to be.. a lot. But I'll take another shot at it.